analysis_options.yaml

```diff
@@ -72,3 +75,12 @@ linter:
     - use_rethrow_when_possible
     - unnecessary_new
 
+dart_code_metrics:
+  rules:
+    - prefer-trailing-comma
+  metrics:
```

Comment:
Note to reviewers: I couldn't find a way for the lib to not check those, so I set the rules accordingly to our current scenario.